tests/0.2.0.json

```diff
@@ -0,0 +1,377 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://recap.build/specs/0.2.0.json/",
+    "type": "array",
+    "items": {
+      "$ref": "#/$defs/RecapType"
+    },
+    "$defs": {
+      "RecapType": {
+        "oneOf": [
+          {
+            "allOf": [
+              {
+                "$ref": "#/$defs/RecapNull"
+              },
+              {
+                "$ref": "#/$defs/RecapCommonFields"
```

Comment:
I chose to put the common fields `alias` and `doc` in its own def instead of repeating them in each of the base types (like `RecapNull` here).

I also chose to put the `allOf` in this `RecapType` `oneOf` union instead of inside each of the base types. I did this both to mirror the DRY way the existing spec introduces these fields, and to emphasize the unique fields when looking at the base types.

Reply:
I ended up repeating myself so that the error messages are easier to read when validation fails.